socorro/external/boto/crashstorage.py

```diff
@@ -54,20 +63,23 @@ def __init__(self, config, namespace="", quit_check_callback=None):
         )
         self.connection_source = config.resource_class(config)
 
+        # Function for calling a function retriably
+        self.retryable = retry(
+            retryable_exceptions=self.connection_source.RETRYABLE_EXCEPTIONS,
+            wait_time_generator=wait_time_generator,
+            module_logger=LOGGER,
+        )
+
     @staticmethod
-    def do_save_raw_crash(boto_connection, raw_crash, dumps, crash_id):
+    def do_save_raw_crash(conn, raw_crash, dumps, crash_id):
```

Comment:
This is cosmetic. `boto_connection` is super long and not overly more informative than just `conn`. Swapping for the shorter name reduces some lines and simplifies things.